posixlib/src/main/scala/scala/scalanative/runtime/SocketHelpers.scala

```diff
@@ -23,7 +23,6 @@ object SocketHelpers {
       var hints = alloc[addrinfo]
       var ret   = alloc[Ptr[addrinfo]]
 
-      libc.memset(hints.rawptr, 0, sizeof[addrinfo])
       hints.ai_family = AF_UNSPEC
```

Comment:
I think that removing the memset is a BUG before/until  PR #2086 is merged. Both here and
a second instance below.  That is, I think the memset is recovery from the `alloc != z.alloc` bug.
There is a slight chance that the memset dates from before (any) alloc cleared memory 
and can be safely removed, slightly improving performance.

Reply:
@LeeTibbert yes, this PR was created before #2086, and I'm still not sure that this one is required as #2086 required.

Reply:
Thanks to refresh this, let me mark it is a draft to save time by anyone.